client/securedrop_client/gui/conversation/export/whistleflow_dialog.py

```diff
@@ -92,7 +94,7 @@ def _show_starting_instructions(self) -> None:
 
     def _send_to_whistleflow(self) -> None:
         timestamp = datetime.datetime.now().isoformat()
-        self._device.whistleflow_export_requested.emit(
+        self._device.send_files_to_whistleflow(
```

Comment:
note - this is a simplification - we're calling the export function directly rather than using signals